### ISSUE TITLE
EFIS System Status Display

### ISSUE THREAD

**Issue by birkelbach** [makerplane/pyEfis#61]
I think it makes sens to add a screen or a widget of some kind for the purpose of displaying system status.  Things like server connection status, database item details, system health like CPU usage, temperatures or whatever makes sense.

**Comment by jmg22** [comment]
Yes, agree with you, plugin  BNO055 have different state, like accuracy. This will be usefull.

**Comment by e100** [comment]
Provided the data is available in fixdb this could be implemented by writing a screenbuilder configuration that user's can include in their screens.
